client/src/locales/en/messages.po

```diff
@@ -999,11 +995,23 @@ msgstr "This portfolio also had an estimated <0>net {changeType, select, gain {g
 
 #: src/components/ViolationsSummary.tsx:12
 msgid "This portfolio has an average of <0>{0}</0> open HPD violations per residential unit."
-msgstr "This portfolio has an average of <0>{0}</0> open HPD violations per residential unit."
+msgstr "This portfolio has an average of <0>{0}</0> open HPD violations per residential unit.<<<<<<< HEAD"
```

Comment:
Also here!

Reply:
so weird, I just looked through everywhere to make sure there are all removed.